.github/workflows/main.yml

```diff
@@ -137,6 +138,9 @@ jobs:
           git config --global user.email "${{ env.GITHUB_SERVICE_EMAIL }}"
           npm run invoke-publish
 
+      - uses: charpeni/turborepo-summary-action@v1
+        if: always()
```

Comment:
Checkout the cool graphs this adds to build summary!: https://github.com/ni/nimble/actions/runs/19976693844